### DISCUSSION TITLE
Activate IAM Access per default

### DISCUSSION THREAD

**Discussion by sbstjn** [superwerker/superwerker#216] [Ideas]
Based on #28, superwerker should enable billing access for IAM users per default.

As superwerker creates a budget report per default, the report should be accessible using SSO users.

https://docs.aws.amazon.com/awsaccountbilling/latest/aboutv2/control-access-billing.html#ControllingAccessWebsite-Activate

Currently, the created budget report is kind of useless, because no one can access it using SSO.

**Comment by s0enke** [comment]
> Based on #28, superwerker should enable billing access for IAM users per default.

IIRC this needs the root user and another Puppeteer script since there is no API for it.

Another possibility would be to list this as another manual step in the living documentation.
